packages/palette/src/elements/Checkbox/Checkbox.tsx

```diff
@@ -58,6 +64,8 @@ export const Checkbox: React.FC<CheckboxProps> = ({
       disabled={disabled}
       error={error}
       onClick={handleClick}
+      tabIndex={0}
```

Comment:
I'm not sure offhand what we do about tabIndexes - will this have any weird tab order effects on pages in force? Is it something that needs to be overrideable?

Reply:
A tabIndex of 0 just means that it's in the natural tab order. -1 takes it out of the tab order (but can be manually focused). And just never use tabIndexes over 0 :) — which jumps the element to the front of the tab order

Reply:
Basically to get something properly accessible when it's not a button: Give it a tabIndex of 0 to bring it into tab order, give it a role (button/checkbox whatever), then add a keypress handler. Here I'm only triggering on spacebar which is what [wai says to do](https://www.w3.org/TR/wai-aria-practices/examples/checkbox/checkbox-1/checkbox-1.html); interestingly I thought Enter would have been supported as well but nope; enter doesn't check native checkboxes either.